tempto-core/src/main/java/com/teradata/tempto/internal/hadoop/hdfs/WebHdfsClient.java

```diff
@@ -183,9 +183,7 @@ public void loadFile(String path, OutputStream outputStream)
         }
     }
 
-    @Override
-    @SuppressWarnings("unchecked")
-    public long getLength(String path)
+    private Object getAttributeValue(String path, String attribute)

```

Comment:
@ebd2 Yes, we can make it public. But then shouldn't we move this to some other class.
@arhimondr - Any comments.